consul/README.md

```diff
@@ -8,10 +8,40 @@ This plan provides the static binary for execution.
 
 ## Usage
 
+### For development:
 ```
 hab svc load core/consul
 ```
 
+### For a cluster:
+On each node,
+```
+hab svc load core/consul --topology standalone --strategy rolling --group leaders
```

Comment:
@fifthecho I believe with hab you can't have a standalone topology that updates with a rolling strategy. If you have tested this functionality by publishing a new version of the package to the builder never mind it didn't work for me previously though.

```suggestion
hab svc load core/consul --topology leader --strategy rolling --group leaders
```

**or**

```suggestion
hab svc load core/consul --topology standalone --strategy at-once --group leaders
```